strings/strings.json

```diff
@@ -7227,5 +7227,28 @@
   "FiltersInAdvancedSearch": {
     "en": "Filters in Advanced Search",
     "fr": "Filtres pour la Recherche Avancée"
-  }
+  },
+  "NoEndpoints": {
+    "en": "has no registered endpoints.",
```

Comment:
The string should accept parameter to format the organization name correctly inside the string.

Otherwise, concatenating string like that might work in french and english, but could be incoherent in chine, for example.

This should become `{0} has no registered endpoints`

And then you can call the translation function like that :  `l('NoEndpoint', organizationName)`